### PR TITLE
Remove leading "$ " from copy-pastable commands

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
When showing both the command and logs, the leading dollar makes things
more readable. However, in snippets with individual commands it doesn't
make it look better, and it's more useful without since it makes the
command directly pasted in the terminal work.

### CODE REVIEW

**Review by jeffwidman — APPROVED** [review]

**Review by jakecoffman — APPROVED** [review]
👍

**Comment by jeffwidman** [comment]
I have a _slight_ preference for retaining the leading `$` for all the reasons @mattt mentions ☝️.

But it's not something I feel strongly on, so thought if someone cared enough to open a PR, they probably care about this more than my level of preference, so that's why I was comfortable approving the PR.

tldr, I'm fine either way, I do slightly lean toward retaining it.

**Comment by deivid-rodriguez** [comment]
No, not really!

I just have the habit of making this contribution whenever I copy a GitHub snippet with the "copy to clipboard" button and end up running a command starting with `$`, I don't think I'm getting more careful with time haha 😅.

I'll just close this given not everybody agrees!

**Comment by deivid-rodriguez** [comment]
Just saw #74. The team seems now more aligned towards favoring easy copy-pasting, so I reopened this!

**Review by jakecoffman — APPROVED** [review]
